src/wombats/constants.clj

```diff
@@ -6,3 +6,28 @@
 (defonce github-scopes "user:email")
 (defonce github-user-profile-url "https://api.github.com/user")
 (defonce github-repo-api-base "https://api.github.com/repos/")
+
+;; Error Codes
+;; Format: a-bb-ccc
+;; a: File type: 0 handlers
+;;               1 dao
+;; b: File number: Increments by one for each file
+;; c: Error number: Increments by one for each error
+(def errors
```

Comment:
Yeah starting to not like the error code structure more and more with each refactor. The goal was to be able to identify where an error was thrown in the codebase by just looking at the number. I guess my counter to "What if an error exists in multiple files?" would be, that should never happen, you should create a new error. With that said, I'm open to ideas for imporvement.